eodag/plugins/download/base.py

```diff
@@ -697,3 +698,36 @@ def download_and_retry(*args: Any, **kwargs: Any) -> T:
             return download_and_retry
 
         return decorator
+
+    def get_assets_values(
```

Comment:
move to `product._assets.AssetsDict.get_values()`

Reply:
done